components/patient-data/ui/src/main/resources/PhenoTips/UIX_Field__date_of_death.xml

```diff
@@ -111,7 +111,22 @@
       <content>{{include document="PhenoTips.PatientSheetMacros" /}}
 
 {{velocity}}
-#__displayFormatted('2-col' 'date_of_death' '' 'date' '')
+#__hasDataToDisplay('date_of_death', $showDod)
+#__hasBooleanDataToDisplay('date_of_death_unknown' $showDodu)
```

Comment:
While "DoB" and "DoD" are common abbreviations, "Dodu" needs at least two more letters to become comprehensible. Please use clearer variable names.